definitions.yaml

```diff
@@ -226,3 +226,16 @@ definitions:
       owner:
         type: string
         description: Name of the organization owning the cluster
+
+  V4CreateUserRequest:
+    type: object
+    required:
+      - password
+    description: Request model for creating a new user
+    properties:
+      password:
+        type: string
+        description: A Base64 encoded password
```

Comment:
Not sure about the base64 encoding, as it's not a solid standard. See also my comments on https://github.com/giantswarm/api-spec/pull/49

Reply:
not sure, but it seems to work quite well in k8s